### PR TITLE
#216 extensions security mirgration

### PR DESCRIPTION
**Pull request by habibimoiz**
#216 - Baton migration script for refactoring aiops references in extensions-security. 
  - Baton migration scripts for any files that need to be renamed.
 - Baton migrations for any Objects that need to be renamed.

### CODE REVIEW

File: extensions/extensions-docker/aissemble-policy-decision-point/src/main/resources/docker/Dockerfile [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
-USER root
-RUN microdnf install jq
-USER 1001
+#This is technically the default if nothing is set, but this ensures if we move it `run-java.sh` will still be able to find it
+ENV JAVA_APP_DIR="/deployments"
 
-RUN mkdir -p /deployments/krausening/
-COPY target/dockerbuild/*.jar /deployments/
+COPY --chown=default ./src/main/resources/truststore/aissemble-secure.jks $JAVA_APP_DIR/
+COPY --chown=default ./src/main/resources/krausening/base/aiops-security.properties $JAVA_APP_DIR/krausening/base/
```
Comment (aaron-gary) [inline extensions/extensions-docker/aissemble-policy-decision-point/src/main/resources/docker/Dockerfile:9]: Q:  I can't remember if you said we were keeping certain property files with the old aiops-* prefix.   Is this one supposed to start with aissemble-* ?

File: extensions/extensions-security/extensions-authzforce/src/main/java/com/boozallen/aissemble/security/authorization/policy/AissembleAttributeProvider.java [hunk 2]
```diff
@@ -121,16 +119,16 @@ public Set<AttributeDesignatorType> getProvidedAttributes() {
      */
     @Override
     public <AV extends AttributeValue> AttributeBag<AV> get(AttributeFqn attributeGUID, Datatype<AV> attributeDatatype,
-                                                            EvaluationContext context, Optional<EvaluationContext> optionalEvaluationContext) throws IndeterminateEvaluationException {
-        String id = attributeGUID.getId();
-        String subject = findSubjectInEnvironmentContext(context);
+        EvaluationContext context, Optional<EvaluationContext> optionalEvaluationContext) throws
+            IndeterminateEvaluationException {String id = attributeGUID.getId();String subject =
```
Comment (aaron-gary) [inline extensions/extensions-security/extensions-authzforce/src/main/java/com/boozallen/aissemble/security/authorization/policy/AissembleAttributeProvider.java:123]: Looks like this command needs a line break(?)

File: foundation/foundation-upgrade/src/test/resources/test-files/v1_8_0/AiopsReferenceJavaMigration/validation/IngestValidated.java [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+
+/**
+ * Performs scaffolding synchronous processing for Ingest. Business logic is delegated to the subclass.
+ *
+ * GENERATED CODE - DO NOT MODIFY (add your customizations in Ingest).
+ *
+ * Generated from: templates/data-delivery-spark/synchronous.processor.base.java.vm
+ */
+public abstract class Ingest extends AbstractPipelineStep {
+
```
Comment (aaron-gary) [inline foundation/foundation-upgrade/src/test/resources/test-files/v1_8_0/AiopsReferenceJavaMigration/validation/IngestValidated.java:67]: The Class name doesn't match the Java file name.  Should this class be named IngestValidated?

**Review by aaron-gary — APPROVED** [review]

File: extensions/extensions-security/extensions-authzforce/src/main/java/com/boozallen/aissemble/security/authorization/policy/AissembleAttributeProvider.java (hunk 2 above)
Comment (habibimoiz) [reply extensions/extensions-security/extensions-authzforce/src/main/java/com/boozallen/aissemble/security/authorization/policy/AissembleAttributeProvider.java]: Corrected this!

File: foundation/foundation-upgrade/src/test/resources/test-files/v1_8_0/AiopsReferenceJavaMigration/validation/IngestValidated.java (hunk 3 above)
Comment (habibimoiz) [reply foundation/foundation-upgrade/src/test/resources/test-files/v1_8_0/AiopsReferenceJavaMigration/validation/IngestValidated.java:67]: Corrected this test file :)

**Comment by habibimoiz** [comment]
Closing out this PR - opening follow PR with resolved git conflicts: https://github.com/boozallen/aissemble/pull/235

**Pull request closed without merging**